#! /bin/sh /usr/share/dpatch/dpatch-run
## 16_vdr-1.3.46_menuitems.c-fix.dpatch by Klaus Schmidinger <Klaus.Schmidinger@cadsoft.de>
## http://article.gmane.org/gmane.linux.vdr/28702
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes set time transponder crash.

@DPATCH@
--- vdr-1.3.46/menuitems.c 2006/04/09 13:10:02     1.36
+++ vdr-1.3.46/menuitems.c 2006/04/10 06:31:02
@@ -572,7 +572,7 @@
      snprintf(buf, sizeof(buf), "%d %s", *value, channel ? channel->Name() : "");
      SetValue(buf);
      }
-  else
+  else if (noneString)
      SetValue(noneString);
 }
